packages/react-vanilla-components/src/components/common/Label.tsx

```diff
@@ -34,7 +34,7 @@ const Label = (props: LABEL) => {
           <button
             className={`${bemBlock}__questionmark`}
             onClick={onClick}
-            aria-label={description}
+            title="Help Text"
```

Comment:
please "aria-label" and pass description in title

Reply:
Added!